src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline

```diff
@@ -717,9 +717,6 @@ public class LibraryServiceSettings extends ServiceApiSettings {
           addCommentsSettings,
           getBookFromArchiveSettings,
           updateBookIndexSettings,
-          streamShelvesSettings,
```

Comment:
Yes, but what change in toolkit is making them disappear? So, how is xsettingsClass.unaryCallSettings changing?